submissions/krylenger/frogger/app.js

```diff
@@ -0,0 +1,246 @@
+const borders = {
+    topY: 49,
+    rightX: 420,
+    bottomY: 464,
+    leftX: 20 
+}
+
+const tileDimensions = {
+    width: 100,
+    height: 83
+}
+
+const score = document.querySelector('.score');
+const lives = document.querySelector('.lives');
+
+const randomSpeed = () => {
```

Comment:
Usually function names should contain a verb (this way it is easier to identify them when reading code). How about `getRandomSpeed`?

Reply:
✅ Done